puppet-lint.gemspec

```diff
@@ -27,7 +27,6 @@ Gem::Specification.new do |spec|
     'Community Contributors',
   ]
   spec.email = [
-    'tim@sharpe.id.au',
     'modules-team@puppet.com',
```

Comment:
is that email address still correct?

Reply:
@pmcmaw is this the one you used?

Reply:
Yes I can confirm this email address is still correct as this is the one I had chatted to Tim via in June.